src/parser.js

```diff
@@ -74,6 +77,11 @@ export function parseCSS (
         vars++
       }
     }
+    if (options.extractStatic) {
+      if (typeof decl.value === 'string' && !/xxx(\d+)xxx/gm.exec(decl.value)) {
```

Comment:
This typeof check might not be needed. I think it only needs to check that the value does not match the replacement pattern.